vcpkg/TOC.yml

```diff
@@ -3,432 +3,434 @@
-    - name: Manual integration
-      href: users/buildsystems/manual-integration.md
-    - name: MSBuild integration
-      href: users/buildsystems/msbuild-integration.md
-  - name: Classic mode
-    href: concepts/classic-mode.md
-  - name: Comparing vcpkg with system package managers
-    href: concepts/system-package-managers.md
-  - name: Continuous Integration
-    href: concepts/continuous-integration.md
-  - name: Default features
-    href: concepts/default-features.md
-  - name: Features
-    href: concepts/features.md
-  - name: GitHub integration
-    href: github-integration.md
-  - name: Helper ports
-    displayName: script port
-    href: maintainers/authoring-script-ports.md
-  - name: Host dependencies
-    displayName: host, dependencies
-    href: users/host-dependencies.md
-  - name: Manifest mode
-    href: concepts/manifest-mode.md
-  - name: Offline Usage
-    displayName: air gap, firewall, proxy, no internet
-    href: concepts/offline.md
-  - name: Open-source registry
-    items:
-    - name: CMake style guide
-      href: contributing/cmake-guidelines.md
-    - name: Maintainer guide
-      href: contributing/maintainer-guide.md
-    - name: PR review checklist
-      href: contributing/pr-review-checklist.md
-  - name: Overlay ports
-    href: concepts/overlay-ports.md
-  - name: Package name resolution
-    href: concepts/package-name-resolution.md
-  - name: Ports
-    href: concepts/ports.md
-  - name: Registries
-    href: concepts/registries.md
-  - name: Supported hosts
-    href: concepts/supported-hosts.md
-  - name: Target platforms
-    items:
-    - name: Supported target platforms
-      href: users/platforms/all.md
-    - name: Android
-      href: users/platforms/android.md
-    - name: MinGW
-      href: users/platforms/mingw.md
-    - name: Universal Windows platform
-      href: users/platforms/uwp.md
-    - name: Windows with MSVC
-      href: users/platforms/windows.md
-    - name: Xbox
-      href: users/platforms/xbox.md
-  - name: Triplets
-    href: concepts/triplets.md
-  - name: Versioning
-    href: users/versioning.concepts.md
+    - name: Asset caching
+      href: concepts/asset-caching.md
+    - name: Binary caching
+      href: users/binarycaching.md
+    - name: Build system integration
+      items:
+        - name: Available build system integrations
+          href: concepts/build-system-integration.md
+        - name: CMake integration
+          href: users/buildsystems/cmake-integration.md
+        - name: Manual integration
+          href: users/buildsystems/manual-integration.md
+        - name: MSBuild integration
+          href: users/buildsystems/msbuild-integration.md
+    - name: Classic mode
+      href: concepts/classic-mode.md
+    - name: Comparing vcpkg with system package managers
+      href: concepts/system-package-managers.md
+    - name: Continuous Integration
+      href: concepts/continuous-integration.md
+    - name: Dependabot support

```

Comment:
The Dependabot entry should be moved under "Default features" to maintain alphabetical order.

Reply:
Done